src/components/FlowRunsBarChart.vue

```diff
@@ -57,6 +57,10 @@
       return null
     }
 
+    if (barsDebounced.value === 0) {
+      return null
+    }
```

Comment:
Unrelated fix. Noticed that before the chart was sized it would sometimes make a request with a limit of `0`. Which of course would return 0 flow runs. So we don't need to make a request for that.